fabric-dimensions-v1/src/main/resources/fabric-dimensions-v1.accesswidener

```diff
@@ -0,0 +1,3 @@
+accessWidener v1 named
+Mutable field net/minecraft/world/dimension/DimensionOptions chunkGenerator Lnet/minecraft/world/gen/chunk/ChunkGenerator;
+Accessible field net/minecraft/world/dimension/DimensionOptions chunkGenerator Lnet/minecraft/world/gen/chunk/ChunkGenerator;
```

Comment:
unnecessary AW an @Accessor would work here

Reply:
If your mutating it might as well use the AW to make it accessible

Reply:
wdym? Accessors can get and set

Reply:
Is `mutable`/`accessible` case sensitive in the spec? Never seen anyone capitalize stuff there

Reply:
It's a bit strange but it ignores the case on them.